src/components/TreeMenu/TreeMenuItem.js

```diff
@@ -7,9 +7,10 @@ const TreeMenuItem = ({
 	level = 0, hasNodes, isOpen,
 	label, searchTerm, openNodes,
 	toggleNode, matchSearch, focused,
-	type, isDisabled, ...props
+	type, isDisabled, resource, ...props
 }) => {
 	const sessionExpired = useSelector(state => state.auth?.sessionExpired);
+	const resources = useSelector(state => state.auth?.resources);
```

Comment:
@aringocode I think resources should be also passed to `<TreeMenu />` as a prop, like we said - resource/resources should be a optional argument. Take a look to the ButtonWithAuthz, it has a similar approach - and why I think its better to do it this way?Because we will loose a dependency to some parts of the application when we will be decoupling the UI's to microfrontend apps